griptape/tools/image_generator/tool.py

```diff
@@ -9,7 +13,16 @@
 
 @define
 class ImageGenerator(BaseTool):
+    """ImageGenerator is a tool that can be used to generate an image.
+
+    Attributes:
+        image_generation_engine: The engine used to generate the image.
+        output_dir: If provided, the generated image will be written to disk in output_dir.
```

Comment:
Should we name this `workdir` for `FileManager` consistency?

Reply:
I can see the argument and don't strongly disagree, but in this case, I think `workdir` is less specific than `output_dir`. That makes sense for `FileManager`, but in this case, we're only outputting files and nothing else.